tiatoolbox/models/architecture/nuclick.py

```diff
@@ -0,0 +1,640 @@
+
+        layers = [conv1]
+
+        if do_batchnorm:
+            layers.append(nn.BatchNorm2d(num_features=out_channels, eps=1.001e-5))
+
+        if activation == "relu":
+            layers.append(nn.ReLU())
+
+        return nn.Sequential(*layers)
+
+
+class MultiscaleConvBlock(nn.Module):
+    """Defines Multiscale convolution block.
+
+    Args:
+        num_input_channels (int):
+            Number of channels in input.
+        num_output_channels (int):
+            Number of channels in output.
+        kernel_sizes (list):
+            Size of the kernel in each convolution layer.
+        strides (int):
+            Size of stride in the convolution layer.
+        use_bias (bool):
+            Whether to use bias in the convolution layer.
+        dilation_rates (list):
+            Dilation rate for each convolution layer.
+        activation (str):
+            Name of the activation function to use.
+
+    Returns:
+        torch.nn.Module:
+            A PyTorch model.
+
+    """
+
+    def __init__(
+        self,
+        num_input_channels: int,
+        kernel_sizes: Union[Tuple[int, int], np.ndarray],
+        dilation_rates: Union[Tuple[int, int], np.ndarray],
+        num_output_channels: int = 32,
+        strides: Union[Tuple[int, int], np.ndarray] = (1, 1),
+        activation: str = "relu",
+        use_bias: bool = False,
+    ):
+
+        super().__init__()
+
+        self.conv_block_1 = ConvBnRelu(
+            num_input_channels=num_input_channels,
+            num_output_channels=num_output_channels,
+            kernel_size=kernel_sizes[0],
+            strides=strides,
+            activation=activation,
+            use_bias=use_bias,
+            dilation_rate=(dilation_rates[0], dilation_rates[0]),
+        )
+
+        self.conv_block_2 = ConvBnRelu(
+            num_input_channels=num_input_channels,
+            num_output_channels=num_output_channels,
+            kernel_size=kernel_sizes[1],
+            strides=strides,
+            activation=activation,
+            use_bias=use_bias,
+            dilation_rate=(dilation_rates[1], dilation_rates[1]),
+        )
+
+        self.conv_block_3 = ConvBnRelu(
+            num_input_channels=num_input_channels,
+            num_output_channels=num_output_channels,
+            kernel_size=kernel_sizes[2],
+            strides=strides,
+            activation=activation,
+            use_bias=use_bias,
+            dilation_rate=(dilation_rates[2], dilation_rates[2]),
+        )
+
+        self.conv_block_4 = ConvBnRelu(
+            num_input_channels=num_input_channels,
+            num_output_channels=num_output_channels,
+            kernel_size=kernel_sizes[3],
+            strides=strides,
+            activation=activation,
+            use_bias=use_bias,
+            dilation_rate=(dilation_rates[3], dilation_rates[3]),
+        )
+
+    def forward(self, input_map):
+        """Logic for using layers defined in init.
```

Comment:
```suggestion
        """Logic for using layers defined in MultiscaleConvBlock init.
```

Please correct this if I am wrong?